### ISSUE TITLE
Add a text alternative for the copyright icon

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#166]
### Problem

[atoms/atoms.copyright](https://a-ui.github.io/core_branding_scss/#atoms-copyright)

The copyright icon has no alternative text. It is inaccessible to screenreaders.

```html
<div class="a-copyright">
  <div class="a-copyright__sign">
    <span class="fa fa-copyright"></span>
  </div>
  <span class="a-copyright__label">Jasper Van Proeyen</span>
</div>
```

![screenshot copyright icon](https://user-images.githubusercontent.com/870049/98798940-c1882800-240e-11eb-8829-30b214020db9.jpg)

### Solution

Suggestion with text-alternative for the copyright icon:

```html
<div class="a-copyright">
  <div class="a-copyright__sign">
    <span class="fa fa-copyright"></span>
    <span class="">copyright </span>
  </div>
  <span class="a-copyright__label">Jasper Van Proeyen</span>
</div>
```
Notice the space at the end of the span to avoid "copyrightJasper Van Proeyen".

### Links and articles

[Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)

**Comment by Veyfeyken** [comment]
Current markup:

```html
<div class="a-copyright">
 <div class="a-copyright__sign">
  <span class="fa fa-copyright">
   <span class="u-screen-reader-only">Copyright </span>
  </span>
</div>
 <span class="a-copyright__label">Jasper Van Proeyen</span>
</div>
```

Suggested markup:

```html
<div class="a-copyright">
 <div class="a-copyright__sign">
  <span class="fa fa-copyright" aria-hidden="true"></span>
  <span class="u-screen-reader-only">Copyright </span>
 </div>
 <span class="a-copyright__label">Jasper Van Proeyen</span>
</div>
```

Difference:
* icon font is hidden
* screenreader text is outside the span of the font icon
